core/templates/dev/head/css/oppia.css

```diff
@@ -1195,11 +1195,34 @@ pre.oppia-pre-wrapped-text {
   border-radius: 4px;
 }
 
+.oppia-editable-section:hover pre {
+	background-color: inherit;
+}
+
+.oppia-editable-section:hover oppia-noninteractive-link,
```

Comment:
I'm a bit worried about this, since it's going to break if new RTE components are added, and developers will not realize they will need to add their components to this list. However, I'm not currently seeing a good way around this.

My first thought was to automatically add an oppia-noninteractive class to all components, but then I realized we store the HTML-with-components in the exploration as a text blob. So doing something like this would require migrating all the content of all explorations, which is complex and a bit error-prone.

Another approach would be to somehow select all tags which have oppia-noninteractive- as a prefix, but I'm not sure CSS supports that...

So, I may need to rescind my previous suggestion, and instead suggest that these RTE-component-specific CSS rules go in the individual RTE component files. Then, if a developer creates a new RTE component, they will see these classes in the other RTE components and would realize they would need to do something similar for theirs. In each case, though, there should be a comment above the rule explaining why it's there.

@wxyxinyu, what do you think? It'd be nice to have a solution that automatically covers all noninteractive components without the need for component-specific CSS at all, but I don't see one. Any creative ideas would be welcome.